mobify/platform-scaffold
# Messaging: Use inline ask instead of default ask

Modifies Merlin's Potions footer newsletter section to use an Inline Ask messaging component, in addition to the existing e-mail signup form - with two Tab sections.

 **JIRA**: [MSG-517](https://mobify.atlassian.net/browse/MSG-517)
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/865

## Changes
- Removes usage of Messaging `DefaultAsk` component from `web/app/containers/app/container.jsx`
- Update footer newsletter section copy and styling as per [MSG-517](https://mobify.atlassian.net/browse/MSG-517)
- Add usage of Messaging `InlineAsk` component

## Todos

## How to test-drive this PR
- Check out the #develop branch of the SDK, and run `npm run prod:build` then `npm link`
- `npm link progressive-web-sdk`
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Scroll to footer, and note the new section
